src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1478,13 +1479,28 @@ public Block get(EnchantItemEvent e) {
 				return e.getEnchantBlock();
 			}
 		}, 0);
+		// Jump Events
 		EventValues.registerEventValue(HorseJumpEvent.class, Entity.class, new Getter<Entity, HorseJumpEvent>() {
 			@Nullable
 			@Override
 			public Entity get(HorseJumpEvent evt) {
 				return evt.getEntity();
 			}
 		}, 0);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, Player.class, new Getter<Player, PlayerEvent>() {
+			@Nullable
+			@Override
+			public Entity get(PlayerEvent evt) {
```

Comment:
Same for any variable that means event
```suggestion
			public Entity get(PlayerEvent event) {
```